reporters_db/data/reporters.json

```diff
@@ -18316,15 +18382,27 @@
             "editions": {
                 "Rodney's Notes": {
                     "end": "1830-12-31T00:00:00",
+                    "regexes": [
+                        "$full_cite",
+                        "$volume_nominative ?$reporter, $page",
+                        "$reporter$page?"
+                    ],
                     "start": "1792-01-01T00:00:00"
                 }
             },
+            "examples": [
+                "Rodney's Notes, 235",
+                "1 Rodney's Notes, 235",
+                "Rodney's Notes"
```

Comment:
This cant be an example.  There is no page

Reply:
I thought the same but the pdf's where comes that reporter doesn't come with a page number, here is an example: https://cite.case.law/del-cas/2/82/

Reply:
interesting.  did you see that its footnoted in source material.

Reply:
im not sure we can handle a citation without volumes and pages

Reply:
> interesting. did you see that its footnoted in source material.

Yes, I saw that is also reporter in Bayard's Notebook, 196

How we should handle this? Create opinionated fix with "Bayard's Notebook, 196" cite instead Rodney's Notes maybe?

Reply:
no no ... I want to keep it in because its accurate but what happens when you feed a citation into our system with just a reporter and no volume/page ...

Reply:
I think I'd argue that we don't care about citations that don't provide a page.

Reply:
What do you think would be the solution for this specific case?

Reply:
Well, I think if we don't have the full citation, we don't have the full citation, so we should just ingest it without a citation or go look it up in a book if we have to.

Reply:
Sounds right to me @mlissner 

Maybe we add a note to the effect that it can be used in old texts that way so that the historical knowledge is there?